### PR TITLE
[DOCs]: NC purge

### PR DESCRIPTION
**Pull request by serinko**
# Description

Documentation update according to #4643

### CODE REVIEW

**Review by tommyv1987 — APPROVED** [review]
CIAO - NC

**Comment by serinko** [comment]
@mfahampshire : Tweaked the socks5 cli page, see if I should add more pictures or given that it's for devs (and NymVPN is the current gig) this suffice.

File: documentation/dev-portal/src/quickstart/socks-proxy.md [hunk 1]
```diff
@@ -12,23 +14,33 @@ If you are using a different operating system, head over to the [Building from S
...
+Use the following command to initialise `nym-socs5-client` where `<ID>` can be anything you want (it's only for local config file storage) and `<PROVIDER>` is suplemented with a NR address:
 
 ```
-./nym-socks5-client init --id quickstart --provider Entztfv6Uaz2hpYHQJ6JKoaCTpDL5dja18SuQWVJAmmx.Cvhn9rBJw5Ay9wgHcbgCnVg89MPSV5s2muPV2YF1BXYu@Fo4f4SQLdoyoGkFae5TpVhRVoXCF8UiypLVGtGjujVPf
+./nym-socks5-client init --id <ID> --provider
 ```
 
-You can always check out the list of endpoints you can use otherwise on the [mixnet service provider explorer page](https://explorer.nymtech.net/network-components/service-providers).
+~~~admonish tip
+Another option to find a NR address associated with a Gateway is to query nodes *Self Described* API endpoint where the NR address is noted like in this example:
```
Comment (mfahampshire) [inline documentation/dev-portal/src/quickstart/socks-proxy.md]: maybe a link here?

**Review by mfahampshire — APPROVED** [review]

**Comment by mfahampshire** [comment]
> @mfahampshire : Tweaked the socks5 cli page, see if I should add more pictures or given that it's for devs (and NymVPN is the current gig) this suffice.

I think this is good!

File: documentation/dev-portal/src/quickstart/socks-proxy.md (hunk 1 above)
Comment (serinko) [reply documentation/dev-portal/src/quickstart/socks-proxy.md]: Will add and publish in the morning if the build looks good.

**Comment by serinko** [comment]
SHIPPING!

**Pull request merged**